lib/renaissance_web/controllers/page_controller.ex

```diff
@@ -1,7 +1,13 @@
 defmodule RenaissanceWeb.PageController do
   use RenaissanceWeb, :controller
+  alias RenaissanceWeb.Helpers.{Auth}
 
   def index(conn, _params) do
-    render(conn, "index.html")
+    case Auth.signed_in?(conn) do
+      true ->
+        render(conn, "index.html")
+      nil ->
```

Comment:
A function that returns `true` or `false` or `nil` is kind of awkward. It should just return `false` for the nil case.

Reply:
Yeah, this is kind of the same comment as my `if` one from before